SouvlakMVP/SouvlakMVP/VerticesConnections.cs

```diff
@@ -245,6 +292,23 @@ public Connection GetConnection(indexT start, indexT stop)
         }
     }
 
+    /// <summary>
+    /// Get indices of uneven vertices of graph
+    /// </summary>
+    /// <returns>List of uneven vertices' indices</returns>
+    public List<indexT> GetUnevenVerticesIdxs()
+    {
+        return this.indexTranslate.Keys.ToList();
+    }
```

Comment:
Should it just be `GetVerticesIndexes()`?
We already add only `oddVertices` (`unevenVertices`) during (re)constructing:
https://github.com/Kri5t0fK/SouvlakiSearch/blob/a7131cca7fe2abab010fea736867ea6c1d2b9661/SouvlakMVP/SouvlakMVP/VerticesConnections.cs#L89 https://github.com/Kri5t0fK/SouvlakiSearch/blob/a7131cca7fe2abab010fea736867ea6c1d2b9661/SouvlakMVP/SouvlakMVP/VerticesConnections.cs#L107

Reply:
Name kept from Graph class and to avoid any confusion